scripts/zones/Crawlers_Nest_[S]/npcs/Tucker.lua

```diff
@@ -8,6 +8,7 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
+    player:startEvent(15) -- rolanberry history dialog
```

Comment:
I'm a bit wary putting this here. Is this his new action after completing the quest? If so, it would be safer to put this in a completed section of the quest and replaceDefault

Reply:
Actually,  I agree with you. I'd like him to have some form of dialog when you talk to him before triggering this quest, else he'll say nothing when you talk to him. However. Player:startEvent may be the wrong dialog command to prompt it.

Reply:
I'm working on a PR right now so that we can handle the 4->5 transition.  4 will be the default action, 5 will replace after that has passed.  We're going to need to be careful with the priority tree for things, but it'll be covered soon.  For now, can we limit the scope of the PR to fixing the trade issue?

If 14/15 happen during the quest, I'm fine with keeping them there though, but not in the NPC script.

Reply:
Alright, I'll remove stuffs from npc file and the quest lua should be all ready to go.